src/internal_events/tcp.rs

```diff
@@ -134,7 +134,7 @@ pub struct TcpEventReceived {
 
 impl InternalEvent for TcpEventReceived {
     fn emit_logs(&self) {
-        debug!(message = "sending event.", byte_size = %self.byte_size);
+        trace!(message = "received event.", byte_size = %self.byte_size);
```

Comment:
…and yet, most of the "received" messages are traces. This obviously should go in for the typo at least.